.docksal/docksal.env

```diff
@@ -1,28 +1,23 @@
-# This is a shared configuration file that is intended to be stored in the project repo.
-# To override a variable locally:
-# - create .docksal/docksal-local.env file and local variable overrides there
-# - add .docksal/docksal-local.env to .gitignore
-
-# Use the default Docksal stack
+### Use the default Docksal stack
 DOCKSAL_STACK=default
 
-# Lock images versions for LAMP services
+### Lock images versions for LAMP services
 # This will prevent images from being updated when Docksal is updated
-#WEB_IMAGE='docksal/web:x.x-apache2.4'
-#DB_IMAGE='docksal/db:x.x-mysql-5.6'
-# Set for compatibility with contrib module that may not be ready for php 7.2
-CLI_IMAGE='docksal/cli:php7.1'
```

Comment:
`CLI_IMAGE='docksal/cli:php7.2'`?

Reply:
nope its below

Reply:
`CLI_IMAGE='docksal/cli:php7.1'` was there for a reason - this was the latest version Drupal 7 core fully supported. 

Looking at [Drupal 7 PHP requirements](https://www.drupal.org/docs/7/system-requirements/php-requirements) - we can use 7.2, but not 7.3 (default in Docksal). So this should be set to `CLI_IMAGE='docksal/cli:php7.2'` + the comment about the version is pinned.

Reply:
Acquia just removed 7.1 and we did run our 7 site just fine on 7.2.